src/site/content/en/animations/animations-overview/index.md

```diff
@@ -84,43 +90,41 @@ For an in-depth look at compositing see the article
 [GPU Animation: Doing It Right](https://www.smashingmagazine.com/2016/12/gpu-animation-doing-it-right/)
 {% endAside %}
 
-## What is a layer?
+## What is a layer? {: #layers }
 
 By placing the things that will be animated or transitioned onto a new layer,
 the browser only needs to repaint those items and not everything else.
-If you have used a package such as Photoshop,
-you will be familiar with the concept of a layer which contains a bunch of elements that can be moved together.
-You can think of these browser rendering layers as being similar to that idea.
+You may be familiar with Photoshop's concept of a layer which contains a bunch of elements that can be moved together.
+Browser rendering layers are similar to that idea.
 
-While the browser does a good job of making decisions about while elements should be on a new layer,
+While the browser does a good job of making decisions about what elements should be on a new layer,
 if it misses one there are ways to force layer creation.
-You can find out about that in the guide to [creating high performance animations](/animations-guide).
-However, creating new layers should be done with care.
-Each layer uses memory,
-on devices with limited memory creating new layers may cause more of a performance problem than the one you are trying to solve.
+You can find out about that in [How to create high-performance animations](/animations-guide).
+However, creating new layers should be done with care because each layer uses memory.
+On devices with limited memory creating new layers may cause more of a performance problem than the one you are trying to solve.
 In addition, each layer's textures need to be uploaded to the GPU.
 Therefore you may well hit constraints of bandwidth between the CPU and GPU.
 
 {% Aside %}
-You can read a good explanation of layers, and how to create them in Surma's article
+You can read a good explanation of layers, and how to create them in
 [Layers and how to force them](https://dassur.ma/things/forcing-layers/).
 {% endAside %}
 
-## CSS vs JavaScript performance
+## CSS vs JavaScript performance {: #css-js }
 
-You might wonder whether it is better from a performance perspective to use CSS or JavaScript for animations.
+You might wonder: is it better from a performance perspective to use CSS or JavaScript for animations?
```

Comment:
The last sentence says `rendering the question moot` but we hadn't actually formulated it as a question